Model/FproCustomer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace monsieurgourmand\Bundle\InterfaceBundle\Model;
+
+
+class FproCustomer extends Master
+{
+    /**
+     * @var integer
+     */
+    private $id;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var integer
+     */
+    private $idFacturationPro;
+
+    /**
+     * @return int
+     */
+    public function getId(): int
```

Comment:
```suggestion
    public function getId(): ?int
```
(un nouvel élément pas encore persisté n'a pas d'ID)